src/main/java/org/sonar/squidbridge/annotations/ActivatedByDefault.java

```diff
@@ -24,6 +24,11 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated since 2.7. Will be removed without alternative.
+ * Rule should be manually enabled when declaring profiles.
```

Comment:
What does "manually" mean? I think we should mention the SQ API which should be used.
Suggestion: "Plugins should define quality profiles with org.sonar.api.server.profile.BuiltInQualityProfilesDefinition"